Add query geo polygon docs (#316)
qdrant/landing_page: qdrant-landing/content/documentation/concepts/filtering.md

If several values are stored, at least one of them should match the condition.
These conditions can only be applied to payloads that match the [geo-data format](../payload/#geo).

#### Geo Polygon

> 

Reviewer: It would be good to have a little introduction explaining why it can be used.
I know the other sections do not have this but let's improve.

Reviewer 2: ```suggestion
Geopolygons search is useful for when you want to find points inside an irregularly shaped area, for example a country boundary or a forest boundary. A polygon always has an exterior ring and may optionally include an interior ring. A lake with an island would be an example of an interior ring. If you wanted to find points in the water but not on the island, you would make an interior ring for the island. 

When defining a ring, you must pick either a clockwise or counterclockwise ordering for your points. 

Currently, we only support unprojected global coordinates (decimal degrees longitude and latitude) and we are datum agnostic

Any points with locations that fall either in or on the boundary of a ring are considered a match .
```

                        lon=60.0,
                        lat=60.0,
                ),
                models.GeoPoint(
                        lon=-70.0,
                        lat=60.0,
                ),
                models.GeoPoint(
                        lon=-70.0,
                        lat=-70.0,
                )
            ]
        ),
        interiors=[
            models.GeoLineString(
                points=[
                    models.GeoPoint(
                            lon=-65.0,
                            lat=-65.0,
                    ),
                    models.GeoPoint(
                            lon=0.0,
                            lat=-65.0,
                    ),
                    models.GeoPoint(
                            lon=0.0,
                            lat=0.0,
                    ),
                    models.GeoPoint(
                            lon=-65.0,
                            lat=0.0,
                    ),
                    models.GeoPoint(
                            lon=-65.0,
                            lat=-65.0,
                    )
                ]
            )
        ]
    )
)
```

> It matches with `location`s inside the given polygon's exterior but not inside any interiors.

Reviewer 2: ```suggestion
A match is considered any point location inside the given polygon's exterior but not inside any interiors.
```

> If several values are stored, at least one of them should match the condition.

Reviewer 2: ```suggestion
If several location values are stored for a point, then any of them matching will include that point as a candidate in the resultset. 
```
